urbanopt/geojson-modelica-translator
# Fix running tests in PyCharm and resolve Spawn paths in system parameters

#### Any background context you want to provide?
The conftest.py file was recently updated in the project which apparently broke the running of unittests within PyCharm. In addition, the system parameters files had non-relative instances of IDF/EPW/MOS files for Spawn. 
#### What does this PR accomplish?
* Fix running py.tests and relative running of tests in PyCharm (or any other relative execution)
* Updated the SystemParameters parsing of files to fully qualify the path upon reading the JSON.
* Add JsonPath (next generation) for use parsing and setting values in JSON/dictionaries using JsonPath (similar to XPath).

#### How should this be manually tested?
* pip install -r requiments.txt
* run `py.tests`
* Go to a specific test in PyCharm and try running the test

Note that `test_ets.py` was not updated as it has different issues and will need to be resolved in another PR (#64).

#### What are the relevant tickets?
#61 

#### Screenshots (if appropriate)
